### ISSUE TITLE
build scripts

### ISSUE THREAD

**Issue by bdharris08** [bdharris08/smarthat#23]
maybe make a build script that would run npm install and cmake on the cpp code

**Comment by easchner** [comment]
Yeah, the CMake stuff is really hacked together at the moment.  I'll make a new Make script here shortly (which I'll need to do anyway to get MRAA up and running), but it may be a few days.
